docker-compose.staging.yml

```diff
@@ -18,3 +18,40 @@ services:
       - ../olsystem:/olsystem
       - ../booklending_utils:/booklending_utils
       - ../olsystem/etc/ia.ini:/home/openlibrary/.config/ia.ini
+      # Persistent volume mount for installed git submodules
```

Comment:
Actually since staging needs all the same stuff as `override`, let's not modify this file and just use `COMPOSE_FILE="docker-compose.yml:docker-compose.override.yml:docker-compose.staging.yml"` for staging.

Reply:
Actually nevermind.